tests/_support/UnitTester.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * Inherited Methods.
+ *
+ * @method void wantToTest($text)
+ * @method void wantTo($text)
+ * @method void execute($callable)
+ * @method void expectTo($prediction)
+ * @method void expect($prediction)
+ * @method void amGoingTo($argumentation)
+ * @method void am($role)
+ * @method void lookForwardTo($achieveValue)
+ * @method void comment($description)
+ * @method void pause()
+ *
+ * @SuppressWarnings(PHPMD)
+ */
+class UnitTester extends Codeception\Actor
```

Comment:
We can use proper namespacing here.

i.e. `use Codeception\Actor` on top of the file and then `class UnitTester extends Actor`